### DISCUSSION TITLE
How many entity attributes should been saved so that next time when I load it from server,  It is at same position, direction, and scale in world space

### DISCUSSION THREAD

**Discussion by 2016Leo123** [bepu/bepuphysics1#42] [General]
How many entity attributes should been saved so that next time when I load it from server,  It is at same position, direction, and scale in world space

**Comment by Gavin-Williams** [comment]
If you want something at same position, then you should save position. Likewise for any other fundamental properties, such as velocity.

**Reply by RossNordby** [reply]
Pretty much- what you should save depends on what you care to persist. Position, orientation, linear velocity, and angular velocity covers kinematic state. Could store a shape id or something to disk to reconstruct/load the entity's shape. Dynamic bodies would need mass/inertia too.

**Reply by 2016Leo123** [reply]
thanks a lot of RossNordby's suggestion
